application/src/test/kotlin/org/gxf/crestdeviceservice/data/convertors/DatabaseFieldEncryptorTest.kt

```diff
@@ -11,6 +12,8 @@ class DatabaseFieldEncryptorTest {
         val expected = "data"
         val encrypted = databaseFieldEncryptor.convertToDatabaseColumn(expected)
         val decrypted = databaseFieldEncryptor.convertToEntityAttribute(encrypted)
-        assertEquals(expected, decrypted)
+
+        assertThat("data").isNotEqualTo(encrypted)
+        assertThat(expected).isEqualTo(decrypted)
```

Comment:
Switch actual and expected values.
```suggestion
        assertThat(encrypted).isNotEqualTo("data")
        assertThat(decrypted).isEqualTo(expected)
```